public/content/documentation/web/page-level/basic-web-page.md

```diff
@@ -77,10 +79,10 @@ Landmarks give structure to the page for the screenreader user to be able to nav
 
 Each page must include:
 
-- Header
-- Nav
-- Main
-- Footer
+- [Header](https://www.magentaa11y.com/#/web-criteria/page-level/header-landmark)
+- [Nav](https://www.magentaa11y.com/#/web-criteria/page-level/navigation-landmark) 
+- [Main]() TODO
```

Comment:
Remove TODO from this and the line below and update the links to main and footer

Reply:
I'm going to leave this for now. I was thinking we can leave TODOS if there are references to components we havent added yet!